themes/luci-theme-material/htdocs/luci-static/material/cascade.css

```diff
@@ -2089,6 +2089,10 @@ label > input[type="radio"] {
 	vertical-align: bottom;
 }
 
+label[data-index][data-depends] {
+	padding-right:2em;
```

Comment:
Please add a space before `2em`, otherwise LGTM :)

Reply:
Thanks for feedback -> Merged